whereAll.ts

```diff
@@ -1,32 +1,37 @@
 import type { Obj, PH, Tests } from './utils/types.ts'
 import curryN from './utils/curry_n.ts'
-import has from './utils/has.ts'
 
 // @types
-type WhereAll_2<T> = ((testObj: Obj<T>) => boolean) &
-  ((testObj?: PH) => WhereAll_2<T>)
+// prettier-ignore
+type WhereAll_2<T> =
+  & ((testObj: Obj<T>) => boolean)
+  & ((testObj?: PH) => WhereAll_2<T>)
 
-type WhereAll_1<T> = ((specs: Tests<T>) => boolean) &
-  ((specs?: PH) => WhereAll_1<T>)
+// prettier-ignore
+type WhereAll_1<T> =
+  & ((specs: Tests<T>) => boolean)
```

Comment:
move this to above line, and remove the `&`.
Take ref [from](https://github.com/Jozty/Fae/blob/v0.6.2/whereAll.ts)

For other functions too